### DISCUSSION TITLE
unexpected termination of runs

### DISCUSSION THREAD

**Discussion by krolbiecki** [MadAnalysis/madanalysis5#257] [General]
Hi,

my runs finish with the following errors:

MA5-WARNING: Error during extracting header info file: local variable 'new_sigma' referenced before assignment
MA5-WARNING: Cannot parse the info file
MA5-WARNING: Info file for cms_exo_20_004 missing or corrupted. Skipping the CLs calculation.
MA5-ERROR: job submission aborted.

(happens for other info files as well)

I run everything on a fresh Ubuntu 22.04, and the latest MA5 from github. 

Additionally, I found an old MA5 v1.9.60 which runs without issues.

Any idea what went wrong? 

Cheers,
Krzysztof

**Comment by BFuks** [comment]
Hi Krzysztof,

Please do a git pull. I have just fixed this. We will update the released tar ball later in the evening. 

Cheers,

Benjamin
